Development/nmos/node_resources.cpp

```diff
@@ -318,9 +319,19 @@ namespace nmos
 
         data[U("media_type")] = value::string(media_type.name);
 
+        if (!color_sampling.name.empty())
+        {
```

Comment:
Would it make sense, you update this PR with the master after the #241 merged, then we can use write
`if (!color_sampling.empty())` instead of
`if (!color_sampling.name.empty())`

Reply:
Yes. I'll rebase and push.